godot-core/src/builtin/rect2i.rs

```diff
@@ -58,7 +62,7 @@ impl Rect2i {
     pub fn from_corners(position: Vector2i, end: Vector2i) -> Self {
         Self {
             position,
-            size: position + end,
```

Comment:
Also this line here was a bug, so should be cherry-picked regardless of this PR's status:

End = Position + Size
-> Size = End - Position